app/assets/stylesheets/pensionsschemefrontend-app.scss

```diff
@@ -10,9 +10,9 @@
 @import 'partials/_shame';
 
 @import 'partials/page-heading';
-@import 'partials/scheme__list';
+// @import 'partials/scheme__list';
 
-@import 'location-autocomplete.min';
+// @import 'location-autocomplete.min';
 
```

Comment:
Wondering what the case is for leaving these commented lines in?

Reply:
when we do the cleanup ticket there's a task to remove the previous assets (js and css) and to only link the new equivalents. to this end the autocomplete css had been moved into the ...twirlupgrade.css file. the 2 views that use autocomplete (established country and address component) have been updated to new fe and merged to 6212 so no pages require that css anymore so we commented it out as a placeholder that this wont concern us. But as the file will be removed at the end it's a moot point. as is this whole explanation really :) but i just wanted to give you some background on the rationale.